features/outbox.feature

```diff
@@ -1,14 +0,0 @@
-Feature: Outbox
```

Comment:
This wasn't working as expected and I wasn't keen on the implementation of the steps. I've decided to cover this functionality in the added unit test as its clearer whats happening / quicker to execute